security-command-center/snippets/src/main/java/vtwo/source/CreateSource.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package vtwo.source;
+
+// [START securitycenter_create_source_v2]
+
+import com.google.cloud.securitycenter.v2.CreateSourceRequest;
+import com.google.cloud.securitycenter.v2.SecurityCenterClient;
+import com.google.cloud.securitycenter.v2.Source;
+import java.io.IOException;
+
+public class CreateSource {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(Developer): Replace the sample resource name
+    // organizationId: Google Cloud Organization id.
+    String organizationId = "{google-cloud-organization-id}";
+
+    createSource(organizationId);
+  }
+
+  public static Source createSource(String organizationId) throws IOException {
+    try (SecurityCenterClient client = SecurityCenterClient.create()) {
+      // Start setting up a request to create a source in an organization.
+      // OrganizationName organizationName = OrganizationName.of(/*organizationId=*/"123234324");
```

Comment:
Same here, why not use `OrganizationName.of(`? Building strings seems a bit odd to me.